run.js

```diff
@@ -0,0 +1,73 @@
+const prompt = require('prompt-sync')();
+const Game = require('./lib/game')
+
+class Run {
+  constructor() {
+    this.game = new Game;
+    this.option = null;
+  }
+
+  printMenu() {
+    console.log("1. Make next roll");
+    console.log("2. Game results")
+    console.log("0. End game"); 
+    this.option = prompt("Please choose an option: ");
+  }
+
+  gameOption(option) {
+    switch (option) {
+      case "1":
+        this.playFrame();
+        break;
+      case "2":
+        this.gameResult();
+        break;
+      case "0":
+        console.log("Thank you for game!")
+        break;
+      default:
+        console.log("Incorrect option\n")
+        break;
+    }
+  }
+
+  playFrame() {
+    console.log('Write result of first roll');
+    let score = prompt();
+    while(! /^\d$|(10)/.exec(Number(score))){
```

Comment:
Great use of while to prevent user incorrectly inputting a score greater than 10